other-docs/guides/migrating/README.md

```diff
@@ -0,0 +1,221 @@
+# Migrating a WordPress Codebase
+
+This guide covers how to migrate a typical WordPress project to Altis.
+
+**Tip:** Throughout this guide, the following conventions are used:
+
+* File names ending with a `/` (such as `wp-admin/`) indicate a directory (folder)
+* Shell commands are indicated by a line starting with `$`, and other lines indicate output. Do not copy the `$`.
+
+
+## Preparing the codebase
+
+### Traditional vs Skeleton
+
+Before you start, identify whether your project is structured in traditional WordPress style, or in WordPress Skeleton style.
+
+In traditional WordPress style, your codebase will look like this:
+
+* `wp-admin/`
+* `wp-content/` (plugins and themes live here)
+* `wp-includes/`
+* `index.php`
+* `wp-config.php`
+* (etc)
+
+In WordPress Skeleton style, your codebase will look like this:
+
+* `content/` (plugins and themes live here)
```

Comment:
Is it worth mention this may also be called 'app/' if it is Bedrock style?

Reply:
Can potentially add in the future, but I would say Bedrock is different enough from Skeleton to be a distinct type here.